### PR TITLE
Routing of RTCP XR packets

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/rtcweb-wg/jsep/issues/523

### CODE REVIEW

**Comment by juberti** [comment]
This seems fine. From #523, I thought the situation was more complicated.

**Comment by aboba** [comment]
@juberti  In SR/RR the sender SSRC and "SSRC of source" is sufficient for routing. XR is similar since there are reports from senders (where the sender SSRC can be used) and receivers (where the "SSRC of source" can be used).  Perhaps Varun can review.

**Comment by fluffy** [comment]
I have not properly reviewed this but I am fine with it being merged it look OK to others.

**Comment by juberti** [comment]
Is this obsoleted by #538?

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -5909,6 +5920,13 @@ a=rtcp-fb:100 nack pli
           <xref target="RFC4585" />, but not previously handled, and the media
           source SSRC for the packet is found in the outgoing SSRC table,
           deliver the packet to the "m=" line associated with that SSRC.</t>
+		
+          <t>If the packet is of type XR, as defined in <xref target="RFC3611" />,
```
Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: Given the bundle does not make XR MTI, I think that this should be prefixed with "If the device supports XR, and ...." ANd make the 3611 ref informative not normative.

**Comment by aboba** [comment]
@juberti Probably.

**Comment by juberti** [comment]
Closing for now.

**Comment by juberti** [comment]
Reopening per discussion in #550. I think this is a better approach as well.

**Comment by juberti** [comment]
LGTM. @fluffy, any concerns before I land this?

**Pull request merged**